src/components/CodeView/index.tsx

```diff
@@ -54,13 +54,13 @@ type Props = PublicProps & RouteComponentProps;
 type RowProps = {
   className: string;
   id: string;
-  ref?: typeof scrollToSelectedLine;
+  ref?: typeof scrollToElement;
 };
 
 export class CodeViewBase extends React.Component<Props> {
   renderWithLinterInfo = ({ selectedMessageMap }: LinterProviderInfo) => {
     const {
-      _scrollToSelectedLine = scrollToSelectedLine,
+      _scrollToElement = scrollToElement,
```

Comment:
Why does this callback need to be applied to the `LinterMessage` ref at the same time it's applied to the table row ref? Aren't scrolling to a line and scrolling to a linter message independent actions?

Reply:
I don't really understand this question, I didn't think I was applying it to both, but I guess that could be possible given the code. In any case, I will probably get rid of this when moving to using Redux state instead.